### PR TITLE
Introduce packaging

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- add MIT license
- use setuptools-scm
- add Makefile target for image and pypi-checking

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Userspace Containerization team at Red Hat
```
Comment (TomasTomecek) [inline LICENSE]: just red hat, we are not a separate enitity :D

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	sudo ansible-bender build --build-volumes $(CURDIR):/src:Z -- ./recipe.yaml $(BASE_IMAGE) $(OGR_IMAGE)
+
+check:
+	PYTHONPATH=$(CURDIR) pytest-3 -v $(TEST_TARGET)
+
+shell:
+	sudo podman run --rm -ti -v $(CURDIR):/src:Z -w /src $(OGR_IMAGE) bash
+
+check-pypi-packaging:
+	sudo podman run --rm -ti -v $(CURDIR):/src:Z -w /src $(OGR_IMAGE) bash -c '\
```
Comment (TomasTomecek) [inline Makefile]: is base_image meant to be used here? won't we run into the same issue you described to me yesterday?

File: recipe.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      - python3-setuptools_scm
+      - python3-setuptools_scm_git_archive
+      - python3-wheel  # for bdist_wheel
+  - name: Install latest twine for sake of check command
+    pip:
+      name:
+      - twine  # we need newest twine, b/c of the check command
+      - readme_renderer[md]
+      state: latest
+  - name: Install dependencies for CI watcher
```
Comment (TomasTomecek) [inline recipe.yaml]: we won't need this task

File: Makefile (hunk 2 above)
Comment (TomasTomecek) [reply Makefile]: just tried this and it works

File: LICENSE (hunk 1 above)
Comment (lachmanfrantisek) [reply LICENSE]: I've copied that from source-git -- we need to change that also there.

Comment (lachmanfrantisek) [reply LICENSE]: It is probably in multiple repos...

File: Makefile (hunk 2 above)
Comment (lachmanfrantisek) [reply Makefile]: The problem was due to these lines:

```

setuptools.setup(use_scm_version=True,
                 packages=setuptools.find_packages(exclude=['tests*']), )
```

I used `exclude=['tests']`, but that was wrong, beacuse it returns e.g. `tests.unit` and `tests.integration`...

Comment (lachmanfrantisek) [reply Makefile]: This needs fix on the sourcegit side as well.

File: recipe.yaml (hunk 3 above)
Comment (lachmanfrantisek) [reply recipe.yaml]: Oh, I missed that. Thanks.

Comment (lachmanfrantisek) [reply recipe.yaml]: But we need most of the packages, so I've changed that only a bit.

**Comment by lachmanfrantisek** [comment]
I've fixed the issues from the review.

File: LICENSE (hunk 1 above)
Comment (TomasTomecek) [reply LICENSE]: yup

**Pull request merged**